src/tests/ftest/server/metadata.py

```diff
@@ -343,9 +387,13 @@ def test_metadata_addremove(self):
         :avocado: tags=ObjectMetadata,test_metadata_addremove
         """
         self.create_pool()
-        if not self.run_dummy_metadata_workload():
-            self.fail("failed to run dummy metadata workload")
-
+        svc_ops_enabled = self.pool.get_property("svc_ops_enabled")
+        if svc_ops_enabled:
+            svc_ops_entry_age = self.pool.get_property("svc_ops_entry_age")
+            if not self.run_dummy_metadata_workload(duration=svc_ops_entry_age):
+                self.fail("failed to run dummy metadata workload")
+        else:
+            self.fail("svc_ops_enabled:0 is not supported for this testcase.")
```

Comment:
This looks good to me. This is to make sure that if the .yaml file is inadvertently changed to specify svc_ops_enabled:0 in the properties: , then it will be prevented from messing up this testcase.